src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -325,37 +372,40 @@ def run_source(self, worker):
         :arg worker: Source worker
         """
 
-        # If task manager is in offline state, do not keep executing sources.
-        while not self.state.should_stop():
-            try:
-                self.logger.info(f"Src {worker.name} calling acquire")
-                data = worker.module_instance.acquire()
-                Module.verify_products(worker.module_instance, data)
-                self.logger.info(f"Src {worker.name} acquire returned")
-                self.logger.info(f"Src {worker.name} filling header")
-                if data:
-                    t = time.time()
-                    header = datablock.Header(self.data_block_t0.taskmanager_id, create_time=t, creator=worker.module)
-                    self.logger.info(f"Src {worker.name} header done")
-                    self.data_block_put(data, header, self.data_block_t0)
-                    self.logger.info(f"Src {worker.name} data block put done")
+        with producers[self.connection].acquire(block=True) as producer:
+            # If task manager is in offline state, do not keep executing sources.
+            while not self.state.should_stop():
+                try:
+                    self.logger.info(f"Source {worker.name} calling acquire")
+                    data = worker.module_instance.acquire()
+                    Module.verify_products(worker.module_instance, data)
+                    self.logger.info(f"Source {worker.name} acquire returned")
+                    self.logger.debug(f"Source {worker.name} produced {data}")
```

Comment:
Can we get a logger.debug noting where we intend to publish the datat (routing key, exchange, etc)?